### PR TITLE
 #2188 HELM 2 support: variant monomers

### PR DESCRIPTION
**Pull request by AliaksandrDziarkach**
Add HELM2 variants support. Add UT

## Generic request
- [ ] PR name follows the pattern `#1234 – issue name`
- [ ] branch name does not contain '#'
- [ ] base branch (master or release/xx) is correct
- [ ] PR is linked with the issue
- [ ] task status changed to "Code review"
- [ ] code follows product standards
- [ ] regression tests updated

### CODE REVIEW

File: core/indigo-core/molecule/src/sequence_saver.cpp [hunk 1]
```diff
@@ -1048,3 +1051,161 @@ void SequenceSaver::saveIdt(KetDocument& doc, std::vector<std::deque<std::string
...
+std::string SequenceSaver::saveHELM(KetDocument& document, std::vector<std::deque<std::string>> sequences)
+{
+    std::string helm_string = "";
+    int peptide_idx = 0;
+    int rna_idx = 0;
+    int chem_idx = 0;
+    using MonomerInfo = std::tuple<HELMType, int, int>;
+    constexpr int polymer_type = 0;
+    constexpr int polymer_num = 1;
+    constexpr int monomer_num = 2;
```
Comment (AlexanderSavelyev) [inline core/indigo-core/molecule/src/sequence_saver.cpp:1087]: what is monomer_num, polymer_num?
is it minimal number of monomers or anything else

Comment (AliaksandrDziarkach) [reply core/indigo-core/molecule/src/sequence_saver.cpp:1087]: Current polymer num and current simple monomer num.

Comment (AliaksandrDziarkach) [reply core/indigo-core/molecule/src/sequence_saver.cpp:1087]: They need to process connection later

Comment (AliaksandrDziarkach) [reply core/indigo-core/molecule/src/sequence_saver.cpp]: Looks like it was used to address tuple, but then tuple was changed to pair.
I removed it.

**Review by AlexanderSavelyev — APPROVED** [review]

**Pull request merged**